app/src/androidTest/java/com/example/android/architecture/blueprints/todoapp/test/chapter11/screens/StatisticsScreen.kt

```diff
@@ -5,6 +5,7 @@ import android.support.test.espresso.action.ViewActions.click
 import android.support.test.espresso.assertion.ViewAssertions.matches
 import android.support.test.espresso.matcher.ViewMatchers.*
 import com.example.android.architecture.blueprints.todoapp.R
+import com.example.android.architecture.blueprints.todoapp.test.chapter3.click
```

Comment:
This import is added but I cannot see where it is used withing this file - is it a mistake?